--- a/node_modules/nx/src/utils/chunkify.js
+++ b/node_modules/nx/src/utils/chunkify.js
@@ -2,7 +2,7 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.chunkify = void 0;
 const child_process_1 = require("child_process");
-const TERMINAL_SIZE = process.platform === 'win32' ? 8192 : getUnixTerminalSize();
+const TERMINAL_SIZE = process.platform === 'win32' ? 8192 : 131071;
 function chunkify(target, maxChunkLength = TERMINAL_SIZE - 500) {
     const chunks = [];
     let currentChunk = [];
